packages/activity/src/index.ts

```diff
@@ -15,23 +15,37 @@
  *
  * ### Cancellation
  *
- * Activity Cancellation serves three purposes:
+ * Activity Cancellation:
  *
- * - It lets an Activity know it doesn't need to keep doing work.
- * - It gives the Activity time to clean up any resources it has created.
+ * - lets the Activity know it doesn't need to keep doing work, and
+ * - gives the Activity time to clean up any resources it has created.
  *
- * Activities may receive Cancellation only if they {@link Context.heartbeat | emit heartbeats} or are Local Activities
+ * Activities can only receive Cancellation if they {@link Context.heartbeat | emit heartbeats} or are Local Activities
  * (which can't heartbeat but receive Cancellation anyway).
  *
- * There are two ways to handle Activity cancellation:
+ * An Activity may receive Cancellation if:
+ *
+ * - The Workflow was Cancelled (via {@link WorkflowHandle.cancel}) and {@link ActivityOptions.cancellationType} was
+ *   **not** set to {@link ActivityCancellationType.ABANDON}.
+ * - The Activity was manually Cancelled (it was called inside a {@link CancellationScope} that was Cancelled via
+ *   {@link CancellationScope.cancel}).
```

Comment:
I'd just say the workflow scope containing the activity was requested to be cancelled (either the root one - e.g. the entire workflow via a handle or a manually created scope).
ABANDON controls whether or not cancellation is sent as an explicit command from the workflow.